### PR TITLE
close #78 by adding links to related repositories to README.md

### LINKED ISSUE #78: documentation: link to related repositories in README.md
To start, I think we should link to `freesound-client` and `react-wavesurfer.js` in `README.md`. Are there any other related repositories we should add?

### PR DESCRIPTION
**Pull request by hamirmahal**
Closes #78 if we choose to merge it.

All this commit does is add

## Related Repositories
[`freesound-client`](https://github.com/amilajack/freesound-client)
[`react-wavesurfer.js`](https://github.com/amilajack/react-wavesurfer.js)

to [`README.md`](https://github.com/amilajack/freesound-player/blob/master/README.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,11 @@
 
 ![freesound-player's appearance after selecting a sound as of 7:55 PM PST Tuesday, September 22, 2020](screenshots/soundpage-2020-09-25-at-8.02.03-PM-PST.png)
 
+## Related Repositories
+[`freesound-client`](https://github.com/amilajack/freesound-client)
```
Comment (amilajack) [inline README.md:14]: We should make these a list with markdown:

```
* [foo](bar)
* [foo](bar)
```

**Review by amilajack — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hamirmahal) [reply README.md:14]: Okay. Will do!

**Review by hamirmahal — COMMENTED** [review]
I'm going to place the related repositories in a list format.

**Review by amilajack — APPROVED** [review]
Good stuff! 🥳

**Comment by hamirmahal** [comment]
Thanks!

**Pull request merged**